include/appimage/desktop_integration/IntegrationManager.h

```diff
@@ -62,7 +68,7 @@ namespace appimage {
              * The map key should be the action identifier and the value the action fields in a plain string i.e.:
              *
              *  ```
-             *  std::map<std::string, std::string> additionalApplicationActions = {{"Remove",
+             *  ApplicationActionList additionalApplicationActions = {{"Remove",
```

Comment:
A vector is much worse, it allows for duplicates. An `unordered_map` is the designated data structure for this task.

Reply:
Good point.

Reply:
Changed to use  `std::unordered_map`. This is a c++11 feature but it should be supported by the gcc-4.8 compiler.